docs/docs/guides/contributing-components.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Contributing Components
+sidebar_label: Contributing Components
+---
+
+:::warning
+
+This functionality is under development. With these docs, we aim to elicit feedback from our community (you!) and focus our development efforts. Currently, we do not offer support for 3rd party components, but we are working toward supporting this.
+
+:::
+
+`Airy Core` is fundamentally a collection of components. When you [install `Airy Core`](../getting-started/installation/introduction.md), you install a set of _core_ components which create the minimum infrastructure to install other components and begin using the platform.
+
+These core components are:
+
+1. airy-controller
+2. api-admin
+3. api-communication
+4. api-websocket
+5. frontend-inbox
+6. frontend-control-center
+
+Airy's real value comes from the additional components you install through our marketplace in the Control Center UI of your Airy Instance. This documentation is on the processes we follow to create, update and store our components.
+
+## Components
+
+Airy is designed to support a plug-and-play architecture. Components are installed and run as pods in your cluster. Components can provide four kinds of functionality:
+
+1. Connector - i.e Rasa, DialogFlow
+2. Source - i.e., Facebook, WhatsApp, SMS
+3. API - i.e
+4. User Interface - i.e
+
+As a technical artifact, each component is a containerized application that is packaged and can be installed. We use Docker and Helm for this.
+
+The Helm package and information on each components (except core components) are stored in an external repository managed by Airy called [airy-components](https://github.com/airyhq/airy-/airy-components). This repository is made up directories where each one contains the component's Helm package and information on it.
+
+## The Component File Structure
+
+Below is a model of the file stucture of a single component inside the [`airy-components`](https://github.com/airyhq/airy-/airy-components) repository.
+
+```
+/airy-components
+	/[COMPONENT_NAME]
+		information.yaml
```

Comment:
`description.yaml` seems a bit more apt. What do you think? 🤔 

Also sharing this neat little app for generating these file graphs: https://tree.nathanfriend.io/

Reply:
Agreed.